_pages/en/capabilities.md

```diff
@@ -4,47 +4,218 @@ title: High Performing IT Capabilities
 ref: it-capabilities
 lang: en
 status: posted
-sections: Archives
+sections: Work In Progress
 permalink: /high-performing-it-capabilities.html
 ---
 
 ## High Performing IT Capabilities
 
-A DevOps Research and Assessment (DORA) [Quick Check of software delivery performance in ESDC and IITB](https://beta.devops-research.com/performance.html?leadtime=1&deployfreq=1&ttr=3&chgfail=4&industry=government) show that we are low performers for deployment frequency, lead time for changes, time to restore service and change fail rate.
+The [State of DevOps Report](https://cloud.google.com/devops/state-of-devops/) has been published annually since 2014, as part of the DevOps Research and Assessment (DORA), a six-year research program that has validated a number of technical, process, measurement, and cultural [capabilities to drive higher software delivery and organizational performance](https://cloud.google.com/devops/#devops-capabilities).
 
-The [State of DevOps Report](https://cloud.google.com/devops/state-of-devops/) has been published annually since 2014, as part of the DORA, a six-year research program that has validated a number of technical, process, measurement, and cultural [capabilities to drive higher software delivery and organizational performance](https://cloud.google.com/devops/#devops-capabilities).
+A [Quick Check of software delivery performance in ESDC and IITB](https://www.devops-research.com/performance.html?leadtime=2&deployfreq=2&ttr=4&chgfail=5&industry=government) show that we are low performers in lead time for changes, deployment frequency, time to restore service and change fail rate.
```

Comment:
recheck with PwC assessment (that was done via interviews with senior management). PwC's assessment contradicts this: page 33 of deliverable Assess Deliverable.

lead time for change, deployment frequency is rated as medium/low
time to restore service is rated as high to low
change failure rate is rated as elite to medium

(this would be at the IITB level though, may not be at the team's level. Each teams could have different rating. I'm thinking DTS and Interop would be very high, but others that depend on SSC INET asking for 45 day turnaround time to deploy in prod would be very low)

Reply:
Current answers used in the example are
- Lead time 1-6 months (Low-medium)
- Deployments 1-6 months (Low-medium)
- Time restore 1-7 days (Medium-high)
- Fail rate 16-30% (Medium-high)

But on the first results table it shows us as Low for all.  Medium and high performers are the same for time to restore.  Medium, high and elite are the same for fail rate.  That's why they mark us as Low.  Looking the the next graph is better.
![image](https://user-images.githubusercontent.com/2353299/84264099-9a8b9a00-aaee-11ea-944d-b49027d91abf.png)

But maybe we leave the link to the quick check, without the results.

Reply:
@smellems 
whatever we say must correlate with the PwC report. If senior managers think we are elite in change failure rate, we cannot say that we are low.

We can also be challenged very easily if we (only the IT strategy team) did the DORA self-assessment on our own without inputs from the other IT teams. Hence the PwC report would have more trusthworthy information.

recommend either:
- Bring the self-assessment to DG-Committee and have the DGs go through it; or
- do not provide any self-assessment results. Instead, explain what DORA thinks "elite" vs "low" means (few days to deploy vs months to deploy); AND where the industry performance is currently.

Reply:
Looking at the results again and they are aligned with the results from PwC.  I added the detailed results